casper/src/main/scala/coop/rchain/casper/BlockCreator.scala

```diff
@@ -193,6 +194,7 @@ object BlockCreator {
       deploys: Seq[DeployData],
       justifications: Seq[Justification],
       maxBlockNumber: Long,
+      sender: PublicKey,
```

Comment:
Since this directly relates to the `sender` field in `BlockMessage`, I feel like `sender` is the better name. Happy to change it if @JosDenmark also prefers "creator" :)

Reply:
I'm indifferent.

Reply:
Sender makes sense in the context of message routing. When discussing blocks in the DAG, for me it doesn't make sense. Anyway, I don't have strong opinions about this